db_creds.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I basically using the same style we had with Sublime-SQL for creds management. In the future, I might add support for directly import BlueLabs db-facts files

Reply:
just realized that although this works when debugging, it's not the recommended. I believe I should be using contribution points configuration to store creds.

Reply:
switched to using vscode settings